test/graphparser.cc

```diff
@@ -677,7 +678,7 @@ void DoConfig() {
     file << "{ \
       \"mjolnir\": { \
       \"id_table_size\": 1000, \
-      \"tile_dir\": \"test/data/parser_tiles\" \
```

Comment:
utrecht was using this? at this point its happened to us so many times that we had directory collisions we might try having the code generate a random suffix for the directories we create :smile:

Reply:
yes.  https://github.com/valhalla/valhalla/blob/master/test/utrecht.cc#L55

Reply:
multiple locations

Reply:
> we might try having the code generate a random suffix for the directories we create

I’d be in favor of that. As long as we print it out it’d be a good time investment. Would fit well in your other test cleanup PR:)

Reply:
Determined that this was not the cause of the failures